teuthology/task/install.py

```diff
@@ -63,6 +63,10 @@
     'python-ceph',
     'rbd-fuse',
 ]
+RHVERSION={}
+RHVERSION['ceph']={
```

Comment:
What's the purpose of this?  Are you just trying to validate that the version put in rhbuild is valid?

If so, maybe just a list of version would be better?